backend/plugins/frontline_api/src/modules/inbox/trpc/inbox.ts

```
@@ -1,39 +1,540 @@
+        } catch (error) {
+          console.error('Error fetching conversations:', {
+            error: error instanceof Error ? error.message : String(error),
+            query,
+          });
+          return {
+            status: 'error',
+            message: 'Failed to fetch conversations',
+            error: error instanceof Error ? error.message : String(error),
+          };
+
+        };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)